### ISSUE TITLE
Custom player model not rendered in portals when gravity is non-default

### ISSUE THREAD

**Issue by Syke-Jr** [qouteall/ImmersivePortalsMod#1245]
### According to your testing, does the issue occur with ONLY Immersive Portals mod and Farbic API?

No

### Describe the issue

I use a popular custom player model mod, MineLittlePony, that only affects the client side. When I disable the mod with its ingame menu, this issue disappears. It happens both in single player and on my server.

When my player gravity is set to anything but the default "down", my player model stops rendering in portals, with some odd edge cases. I will do my best to document them.

I am playing on the latest 1.19.2 version and using the Quilt version of Gravity API. I'm willing to update to 1.19.3 but only if I know it's going to solve the issue.

Video [edge cases shown starting at 0:40]:
https://www.youtube.com/watch?v=UknbwzhNAu8

### Post the log here
https://pastebin.com/kb0dKhnq

**Comment by qouteall** [comment]
It may be an issue of gravity api. Is the third person view normal when the gravity is not default?

**Comment by Syke-Jr** [comment]
> It may be an issue of gravity api. Is the third person view normal when the gravity is not default?

yes, when gravity is default everything works perfectly. The strange part is that the portal *does* render the player model in nondefault gravity, just only some of the time.

**Comment by qouteall** [comment]
Is the third person view normal when the gravity is not default? I need to know whether it's related to immptl.

**Comment by Syke-Jr** [comment]
> Is the third person view normal when the gravity is not default? I need to know whether it's related to immptl.

The third person view in non default gravity is shown in the video-- I'm not sure what you're asking, sorry. When gravity is not default, third person view works fine but the player model is still not rendered all the time in portals. The last part of the video shows this; the behavior is identical to first person.

**Comment by qouteall** [comment]
Sorry I didn't see the video

**Comment by qouteall** [comment]
It's a complex 3-mod compatibility issue
